dp0.tex

```diff
@@ -56,13 +56,38 @@ \subsubsection{Data Products Offered}
+Specifically, in DP0.1, the following tables from DESC will be provided as one database \texttt{dp01\_dc2\_catalogs} in one TAP schema:
+\begin{itemize}
+\item \texttt{position}: DESC internal data as ingested in DESC's Qserv instance at IN2P3.
+\item \texttt{dpdd\_ref}: DESC internal data as ingested in DESC's Qserv instance at IN2P3.
+\item \texttt{dpdd\_forced}: DESC internal data as ingested in DESC's Qserv instance at IN2P3.
+\item \texttt{object}: DESC internal data v2 at NERSC provided to us in parquet format.
+\item \texttt{truth\_match}: DESC internal data v2 at NERSC provided to us in parquet format.
+\end{itemize}
+
+The science data products in the Butler Gen3 repository depend on the availability in the DESC-provided Gen2 data repository and the Gen2-to-3 conversion.
+Besides raw data, the following 4 reruns were obtained from DESC's copy at IN2P3.
+\begin{itemize}
+\item \texttt{run2.2i-calexp-v1} (Gen2) \texttt{2.2i/runs/DP0.1/calexp/v1} (Gen3)
+\item \texttt{run2.2i-coadd-wfd-dr6-v1-u} (Gen2) \texttt{2.2i/runs/DP0.1/coadd/wfd/dr6/v1/u} (Gen3)
+\item \texttt{run2.2i-coadd-wfd-dr6-v1-grizy} (Gen2) \texttt{2.2i/runs/DP0.1/coadd/wfd/dr6/v1/grizy} (Gen3)
+\item \texttt{run2.2i-coadd-wfd-dr6-v1} (Gen2) \texttt{2.2i/runs/DP0.1/coadd/wfd/dr6/v1} (Gen3)
+\end{itemize}
+
+Data were transferred from IN2P3 to NCSA, and converted into a Gen3 repository at NCSA.
+The repository provided on IDF is based on the DC2 repo at NCSA on March 16.
+Newer data ingested into NCSA's repo will not be ported to IDF.
+Some dataset types, such as warps, will not be provided.
+See Appendix \ref{sec:dp01butler} for a full list of expected dataset types in DP0.1.
+
+
+In DP0.2, the exact science data products depend on what pipelines are ready for our reprocessing.
 
 Questions:
 
 \begin{itemize}
 
-\item Are we offering parquet files? --- No promise. Currently our SDMified parquet-generating pipelines are HSC only and Gen2 only. If parquet files are offered the access will be via the read-only Butler Gen3 repo.
+\item Are we offering parquet files? --- No in DP0.1; likley yes in DP0.2. Currently our SDMified parquet-generating pipelines are HSC only and Gen2 only. If parquet files are offered the access will be via the read-only Butler Gen3 repo.
 
 \item We should presumably explicitly rule out bulk download  --- YES. However, this ({\it was} discussed at the 2019 PCW, as a potential mitigation against there not being batch compute available in DP0. If a particular group requested bulk download, it could be an opportunity to start developing that capability. We will also need to know whether to allow DESC bulk download access as part of the MOU to gain access to DC2: they may well want to download all the re-processed products, for their own purposes (and to develop their capability to ingest and work with bulk downloads).
 
```

Comment:
I would feel more comfortable if this paragraph definitively ruled it out for DP0.1.  It could still be left as TBD for DP0.2.

Reply:
I didn't write this paragraph but I interpreted the capital "YES" as definitely ruling it out for DP0. Do you mean you want the statement to be even stronger?

Reply:
Well, it's kind of stream-of-consciousness the way it reads.  "YES." followed by "However," sounds a lot like "Well, not really" to me.

Maybe "We have ruled out offering bulk download facilities for DP0.1.  They are unlikely to be available at any time during the DP0 era, but we are still evaluating that against the time scale of the Data Previews and the schedule for the remaining required development effort.  Making bulk download available had previously been discussed as a possible mitigation for the absence of user-facing batch compute in DP0.  We still need to determine whether we need to provide some form of bulk access to DESC to return the Rubin-processed DC2 data from DP0.2 to them for their own use, both for scientific purposes and to allow them to develop their capability to ingest and work with bulk downloads."

@leannep Any thoughts on this?

Reply:
I see what you mean now. What you suggested sounds clearer and it's in the commit now.

Reply:
Just added a comment above before seeing this, essentially to the same effect. I personally would leave all bulk download until DP1. (i.e LSST data). DESC DC2 data are now public and those who want to bulk download can go to the DESC portal. We have plenty of time to test bulk download from DP1 onwards.  We are planning to provide batch compute in DP0.2 I think?

Reply:
No, we won't provide user batch compute in DP0.2.